shell_provisioner/sylius/create_for_contribution.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd /var/www/sites/sylius
+
+composer install
```

Comment:
Might be good to add `--no-progress` also here to save output.

And what about adding `--prefer-source`? Isn't that required to get the repositories inside vendor/?

Reply:
"--prefer-source: There are two ways of downloading a package: source and dist. For stable versions Composer will use the dist by default. The source is a version control repository. If --prefer-source is enabled, Composer will install from source if there is one. This is useful if you want to make a bugfix to a project and get a local git clone of the dependency directly."

@lchrusciel what composer command do you recomand me ?

Reply:
I would vote for `--no-progess` only. `--prefer-source` would be useful if we would like to fix sylius dependencies from vendor folder.